draft-zip-xxx-wallet-eos-eol.rst

```diff
@@ -0,0 +1,377 @@
+  phrases)
+
+
+Abstract
+========
+
+Zcash Wallet Applications might no longer be maintained, supported or serviced by
+their maintainter from a point in time onwards. This ZIP defines a set of best practices
+and recommendations to minimize the impact of the End of Service of a Zcash wallet 
+application onto its current or potential users.
+
+
+Motivation
+==========
+
+Cryptocurrency peer-to-peer networks are designed to be long-lived decentralized systems
+that should tolerate the pass of time. While it is considered that the network implementing 
+these decentralized protocols would live indefinitely, wallet applications might be subject to
+shorter lifecycles. There are many reasons for them to reach a point in time where they will
+longer able to keep functioning. Whatever the reason might be, this document considers that 
+the result for end-users is indistinct: they will not be able to carry on their usual activities
+on the application once the EOL/EOS day comes. To that end, this ZIP will discuss the topic of
+sunsetting end-user applications gracefully by anticipating the moment as an invariant of any
+product's lifecycle. 
+
+
+Requirements
+============
+
+This document aims to ensure the ability for user to exercise complete sovereign self-
+custody of their keys and funds related to them. Although it will refer specifically to 
+non-custodian wallets (or similar applications that handle Zcash keys), it can also apply to
+custodian applications reaching EOS/EOL. The goal of this ZIP is to ensure that the user is in 
+control of its funds (or access to them) at all times, and that they can be provided ways to
+keep that access beyond a specific product EOS/EOL in a "hassle-free" and user-friendly way.
+Maintainers SHOULD always be eagerly looking to provide well-tested, dependable and mission
+critical use cases that produce an application feature that can be made available to users in 
+the case of an EOS/EOL scenario.
+
+
+Non-requirements
+================
+
+Wallets that are considered to be "in testing", "alpha", "beta" or "developer preview"
+programs and openly adversiting that to their users, therefore, not considered to be deployed 
+to an open audience as a finished product that is intended for end-users MAY opt not to follow 
+this practices. Althought, they SHOULD consider implementing them as part of their process to 
+delivering the application into production.
+
+
+Specification
+=============
+
+What is an EOS/EOL event?
+-------------------------
+End of Life / End of Service: refers to a point in time where the wallet
+software will no longer be actively maintained or developed, therefore its
+operation cannot be guaranteed anymore or terminated indefinitely.
+
+
+Scope of an EOS/EOL event
+-------------------------
+
+The breath of an application userbase and ecosystem may vary depending of the number of Operating
+Systems, Platforms and/or architectures it supports. 
```

Comment:
Terms should only be capitalized if it's usual to do so, or if they're defined in Terminology and the document capitalizes all such terms. The latter is optional.